openstack/keppel-anycast-dashboard/templates/deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       enableServiceLinks: false
       containers:
         - name: nginx
-          image: {{ .Values.global.registry | required ".Values.global.registry not found" }}/shared-app-images/alpine-nginx:mainline-slim-latest
+          image: {{ include "shared-app-images.ref" (tuple $ "alpine-nginx" "mainline-slim") }}
           imagePullPolicy: Always
```

Comment:
Please remove `imagePullPolicy: Always` on all containers that had their image adjusted. This policy is only relevant when using a mutable tag like `latest`. For immutable tags (i.e. tags that never change after being pushed), the default of `imagePullPolicy: IfNotPresent` is preferable.